utils/Utils.mjs

```diff
@@ -11,7 +12,7 @@ const SELF_DESTRUCT_TIMEOUT = 45000;
 export const sendMessageWithOptions = async (message, messageBody,
   selfDestruct = false,
   selfDestructTimeout = SELF_DESTRUCT_TIMEOUT) => {
-  message.channel.send(messageBody).then((msg) => {
+  return message.channel.send(messageBody).then((msg) => {
```

Comment:
This function previously was either returning true if selfDestruct, or returning the msg that was sent via the .then(). It might do the same thing, but I don't know well enough to dispute it. So with this additional return, is it still maintaining the same old functionality?

Reply:
sendMessageWithOptions was previously returning undefined no matter what. message.channel.send().then() returns the message object or true if selfDestruct, but sendMessageWithOptions wasn't doing anything with it.